data/sql/updates/pending_db_world/rev_1718974840446011698.sql

```diff
@@ -0,0 +1,166 @@
+--
+UPDATE `creature_template` SET `flags_extra` = `flags_extra` | 524288 WHERE `entry` IN
+(10184, -- Onyxia
+15928, -- Thaddius
+15931, -- Grobbulus
+15932, -- Gluth
+15936, -- Heigan the Unclean
+15952, -- Maexxna
+15953, -- Grand Widow Faerlina
+15954, -- Noth the Plaguebringer
+15956, -- Anub'Rekhan
+15989, -- Sapphiron
+15990, -- Kel'Thuzad
+16011, -- Loatheb
+16028, -- Patchwerk
+16060, -- Gothik the Harvester
+28859, -- Malygos
+28860, -- Sartharion
+30449, -- Vesperon
+30451, -- Shadron
+30452, -- Tenebron
+31125, -- Archavon the Stone Watcher
+32845, -- Hodir
+32857, -- Stormcaller Brundir
+32865, -- Thorim
+32867, -- Steelbreaker
+32871, -- Algalon the Observer
+32906, -- Freya
+32927, -- Runemaster Molgeim
+32930, -- Kologarn
+33113, -- Flame Leviathan
+33118, -- Ignis the Furnace Master
+33186, -- Razorscale
+33271, -- General Vezax
+33288, -- Yogg-Saron
+33293, -- XT-002 Deconstructor
+33432, -- Leviathan Mk II
+33515, -- Auriaya
+33651, -- VX-001
+33670, -- Aerial Command Unit
+33993, -- Emalon the Storm Watcher
+34441, -- Vivienne Blackwhisper
+34444, -- Thrakgar
+34445, -- Liandra Suncaller
+34447, -- Caiphus the Stern
+34448, -- Ruj'kah
+34449, -- Ginselle Blightslinger
+34450, -- Harkzog
+34451, -- Birana Stormhoof
+34453, -- Narrhok Steelbreaker
+34454, -- Maz'dinah
+34455, -- Broln Stouthorn
+34456, -- Malithas Brightblade
+34458, -- Gorgrim Shadowcleave
+34459, -- Erin Misthoof
+34460, -- Kavina Grovesong
+34461, -- Tyrius Duskblade
+34463, -- Shaabad
+34465, -- Velanaa
+34466, -- Anthar Forgemender
+34467, -- Alyssia Moonstalker
+34468, -- Noozle Whizzlestick
+34469, -- Melador Valestrider
+34470, -- Saamul
+34471, -- Baelnor Lightbearer
+34472, -- Irieth Shadowstep
+34474, -- Serissa Grimdabbler
+34475, -- Shocuul
+34496, -- Eydis Darkbane
+34497, -- Fjola Lightbane
+34564, -- Anub'arak
+34780, -- Lord Jaraxxus
+34796, -- Gormok the Impaler
+34797, -- Icehowl
+34799, -- Dreadscale
+35013, -- Koralon the Flame Watcher
+35144, -- Acidmaw
+36597, -- The Lich King
+36612, -- Lord Marrowgar
+36626, -- Festergut
+36627, -- Rotface
+36678, -- Professor Putricide
+36855, -- Lady Deathwhisper
+36899, -- Big Ooze
+36939, -- High Overlord Saurfang
+36948, -- Muradin Bronzebeard
+37562, -- Gas Cloud
+37813, -- Deathbringer Saurfang
+37955, -- Blood-Queen Lana'thel
+37970, -- Prince Valanar
+37972, -- Prince Keleseth
+37973, -- Prince Taldaram
+38216, -- Mutated Professor Putricide
+38433, -- Toravon the Ice Watcher
+38883, -- Idle Before Scaling
+39231, -- The Lich King (Temp)
+39625, -- General Umbriss
+39788, -- Anraphet
+39863, -- Halion
+40142, -- Halion
+40177, -- Forgemaster Throngus
+40484, -- Erudax
+40960, -- Bug 193702
+41270, -- Onyxia
+41376, -- Nefarian
+41378, -- Maloriak
+41442, -- Atramedes
```

Comment:
I think every creature below this ID (41442) does not exist in AC's `creature_template` table.

Reply:
41270 does not exist in acore; neither in TDB. Maybe they did at some point, need to check the list again to remove non-existing entries